### PR TITLE
minor readme changes

### PR DESCRIPTION
**Pull request by dsweber2**
### Checklist

Please:

- [X] Make sure this PR is against "dev", not "main" (unless this is a release
      PR).
- [X] Request a review from one of the current epidatr main reviewers:
      brookslogan, dshemetov, nmdefries, dsweber2.
- [ ] Makes sure to bump the version number in `DESCRIPTION`. Always increment
      the patch version number (the third number), unless you are making a
      release PR from dev to main, in which case increment the minor version
      number (the second number).
- [ ] Describe changes made in NEWS.md, making sure breaking changes
      (backwards-incompatible changes to the documented interface) are noted.
      Collect the changes under the next release number (e.g. if you are on
      1.7.2, then write your changes under the 1.8 heading).

### Change explanations for reviewer

sorry for the spam. First this, then the merge to main. should be submitting to CRAN after that though

### Magic GitHub syntax to mark associated Issue(s) as resolved when this is merged into the default branch

- Resolves #{issue number}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ pak::pkg_install("cmu-delphi/epidatr@dev")
 ```
 
 Our CRAN listing is
-[here](https://cran.r-project.org/web/packages/epidatr/index.html).
+[here](https://CRAN.R-project.org/package=epidatr/index.html).
```
Comment (nmdefries) [inline README.md:85]: question: Looks like this link redirects to the old URL. Is there a benefit to doing it this way?

**Review by nmdefries — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dsweber2) [reply README.md:85]: cran prefers this format, for whatever reason

Comment (dsweber2) [reply README.md:85]:
```

Found the following (possibly) invalid URLs:
  URL: https://cran.r-project.org/web/packages/epidatr/index.html
    From: README.md
    Status: 200
    Message: OK
    CRAN URL not in canonical form
  URL: https://www.cdc.gov/datastatistics/index.html
    From: README.md
    Status: 404
    Message: Not Found
  The canonical URL of the CRAN page for a package is 
    https://CRAN.R-project.org/package=pkgname
```

Comment (dshemetov) [reply README.md:85]: Huh, I guess it's kinda neat that it checks URLs for you.

Comment (dshemetov) [reply README.md:85]: [This page](https://stackoverflow.com/questions/64257455/cran-url-not-in-canonical-form) says the canonical URL is 

`https://cran.r-project.org/package=package-name`

?

Comment (dsweber2) [reply README.md:85]: `urlchecker::url_check()` was satisfied, which gives a more obscure version of the win_builder result, so I think we're good.

**Pull request merged**